public/js/serchDir/searchDir.js

```diff
@@ -0,0 +1,83 @@
+const affiliateId = 789876;
```

Comment:
Here is the modified `searchDir.js` file. 

This makes a request to the API to get a list of the available domains, and then displays them. I would suggest moving a bulk of this logic to the server.